src/mpid/ch4/subconfigure.m4

```diff
@@ -189,10 +189,10 @@ AC_ARG_ENABLE(ch4-shm,
        level:
          no        - Do not build or use CH4 shared memory.
          yes       - Build CH4 shared memory, but do not use it by default (Your chosen netmod must provide it).
-         exclusive - Build and exclusively use CH4 shared memory. (Default)
+         exclusive - Build and exclusively use CH4 shared memory.
```

Comment:
Seems like now users lost an easy way to tell which one is the default. Can we say like "default for UCX / default for others"?